src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/WebJobs.Logging.ApplicationInsights.csproj

```diff
@@ -25,17 +25,17 @@
   
   <ItemGroup>
   <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.8.2" />
```

Comment:
I would recommend upgrading AspNetCore to v2.12.0 to remain in sync with the rest of the SDK.